src/DrCabal/Cli.hs

```diff
@@ -29,8 +31,9 @@ newtype WatchArgs = WatchArgs
     { watchArgsOutput :: FilePath
     }
 
-newtype ProfileArgs = ProfileArgs
+data ProfileArgs = ProfileArgs
     { profileArgsInput :: FilePath
+    , profileStyleArgsInput :: Style
```

Comment:
Let's rename the record field for the sake of consistency with the naming schemes
```suggestion
    , profileArgsStyle :: Style
```

Reply:
Done.